web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -51,101 +59,231 @@ const useStyles = makeStyles({
     }
   },
   tableHeader: {
-    fontWeight: 'bold'
+    fontWeight: 'bold',
+    padding: '1px'
+  },
+  borderless: {
+    border: 'none'
+  },
+  controls: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'baseline'
   }
 })
 
 const DailyViewTable = (props: Props) => {
   const classes = useStyles()
-  const { fireCentres } = useSelector(selectHFIStations)
-  const stationDataLoading = useSelector(selectHFIStationsLoading)
+
+  const roundToOneDecimalPlace = (value: number | undefined): number | undefined => {
```

Comment:
Should we not rather use the built in `toFixed` function?

Reply:
Artifact from: https://github.com/bcgov/wps/pull/1045

But can address.